mandatory/4-tax.js

```diff
@@ -5,7 +5,11 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  let salesTax = price * 0.2;
```

Comment:
Hello, I believe an error was made on this line. * means to multiply. I believe yous should divide price by 0.2 and not multiply.